### ISSUE TITLE
We need to determine how we want to handle CSS 

### ISSUE THREAD

**Issue by rpearce** [rpearce/react-medium-image-zoom#165]
We need to figure out how we want to handle styles.

Currently, we have CSS Modules & PostCSS working in Rollup.js to give us styles that get injected at the top of the `<head>`.

### Problem
If sites have a strict content security policy (CSP), then dynamically adding a `<style>` tag to the page with unique ID-based content will cause this to not work at all.

### Solutions
Both of these solutions involve outputting a CSS file that _must_ be imported by the consumer.

* Should we be going the route of `react-bootstrap` and merely have a single `.css` file with static values that can be overriden by the consumer?
* Should we use rollup to output a CSS file and give our component an object mapping to what the classes are available so it still feels like CSS Modules?
* Should we be like [react-select](https://github.com/JedWatson/react-select/tree/master/packages/react-select) and go with [emotion](https://emotion.sh/docs/introduction)?
* ???

### My Opinion
I'm currently leaning towards doing the same thing `react-bootstrap` does and have the CSS file be importable via the package but also available via a CDN.

* * *

Tangentially related issue: https://github.com/rpearce/react-medium-image-zoom/issues/147

**Comment by rpearce** [comment]
cc @spencerfdavis for more thoughts on this

**Comment by rpearce** [comment]
cc @matthewsecrist if you get any spare time and have any thoughts on the matter!

**Comment by rpearce** [comment]
My gut feeling right now is to see how it feels having rollup output a CSS file that is imported by a consumer, and if that doesn't feel great, just have a plain old CSS file and classes

**Comment by rpearce** [comment]
This bug is quite annoying at the moment: https://github.com/egoist/rollup-plugin-postcss/issues/70. It's also feeling like a number of these rollup plugins aren't maintained, either.

**Comment by rpearce** [comment]
I'm going to look at potentially replacing rollup with https://parceljs.org. TBD.

Otherwise, I am going to just use straight CSS, run it through PostCSS and dump it as part of the `build` in the `dist` folder as `styles.css`.

**Comment by rpearce** [comment]
See https://github.com/rpearce/react-medium-image-zoom/pull/184 for updates.

tl;dr => sticking with rollup & switching to generating a `styles.css` file from our CSS modules.
